gammapy/data/observations.py

```diff
@@ -359,6 +382,31 @@ def check_observation(self):
 
         return messages
 
+    def to_observation_cta(self):
+        """
+        Convert a `~gammapy.data.DataStoreObservation` to an `~gammapy.data.ObservationCTA`
```

Comment:
Just put this: "Convert to `~gammapy.data.ObservationCTA`".
It's clear that it's converting from a DataStoreObservation, since this is a method on that class.
(best to not repeat class names too much, because it's harder to maintain when they change)

Reply:
Done